src/components/Image/Image.tsx

```diff
@@ -7,6 +7,16 @@ import { Accessibility } from '../../lib/accessibility/interfaces'
 
 /**
  * An image is a graphic representation of something.
+ * @accessibility
+ * Default behavior: ImageBehavior
```

Comment:
just as a matter of thought: for the future we might consider to introduce `Default behavior` section under `@` symbol, so that it could be parsed by parsing tools and segregated from the rest of content

Reply:
yes, we can think about that in next PR

Reply:
just a note to consider - in that case we would need to maintain the information about the default behavior in two places - in the comments and in defaultProps.

Reply:
yes, but this is something that we have even now, so the situation won't be any different. But this is a very good point raised - probably we should utilize the fact that defaultBehavior is already defined, so that there won't be any chance for these two different places of declaring it to come out os sync at some point